File: Thehana/app/src/main/java/com/byapps/thehana/util/ProgressUtil.java
package com.byapps.thehana.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.byapps.thehana.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2025/VicLive/app/src/main/java/com/creator/viclive/util/ProgressUtil.kt
package com.creator.viclive.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.viclive.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}